tests/BuildingSearchTests.py

```diff
@@ -0,0 +1,104 @@
+import unittest
```

Comment:
Should we start using a continuous integration tool in github to run tests as well as enforce code style?

Reply:
Alex set up Travis!